scipy/stats/_continuous_distns.py

```diff
@@ -6384,6 +6384,38 @@ def _munp(self, n, c):
     def _entropy(self, c):
         return np.log(2.0/c) + 1.0
 
+    @_call_super_mom
+    def fit(self, data, *args, **kwds):
+        data, fc, floc, fscale = _check_fit_input_parameters(self, data,
```

Comment:
The code style you suggest is more modern and maybe popular. I just followed the code style of the existing scipy code base, which is more compact though less uniform looking.

Reply:
In cases like this either is fine - style is not uniform across the repo and we allow for author preference to a reasonable extent (i.e. we don't spend review time on it).

_Hopefully one day we can get an auto-formatter, but there are challenges with that and for now it's fine and it would not be a good idea to start nitpicking style in every PR_